### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.1.1][IntelliJ][Wipro]: Buttons in Azure Explorer- Open ACR Explorer  dialog are not accessible with keyboard.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1445]
**Repro Steps:**

1. In "Azure Explorer" , select the pushed image under container registries and use shift+F10 to open the context menu.
2. Navigate to and select "Open ACR Explorer" menu item .
3. Azure Explorer- Open ACR Explorer  window will open.
4. Navigate and try to access the  "Properties" Expand collapse button ,"Enable" ," Disable"," Copy to clipboard "," Refresh"," Up and down" buttons with keyboard.

**Expected Result:**
 "Properties" Expand collapse button ,"Enable" ," Disable"," Copy to clipboard"," Refresh"," Up and down" buttons  should be accessible with keyboard in Azure Explorer- Open ACR Explorer window.

**Actual Result:**
 "Properties" Expand collapse button ,"Enable" ," Disable"," Copy to clipboard"," Refresh"," Up and down" buttons  are not accessible with keyboard in Azure Explorer- Open ACR Explorer dialog

**Note:** unable to navigate with keyboard in Azure Explorer- Open ACR Explorer dialog.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![buttons not accessible in acr window](https://user-images.githubusercontent.com/22956130/35729622-30a772aa-07c4-11e8-8880-5a8de9ca0ad6.png)

**Comment by jdneo** [comment]
Reference:
https://youtrack.jetbrains.com/issue/IDEA-191315

**Comment by Eskibear** [comment]
External issues tracked in #1627

**Comment by jdneo** [comment]
* **Importance of experience**: minimal. Very low usage based on telemetry
* **Quality of experience**: not impacted. Users can do the same operations in Azure Portal. 
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by jdneo** [comment]
Workaround:

The same operations can be achieved in Azure Portal:
![image](https://user-images.githubusercontent.com/6193897/43813104-cde1d23e-9af6-11e8-893f-2a7c3b56343c.png)


![image](https://user-images.githubusercontent.com/6193897/43813011-78711bfc-9af6-11e8-830d-fa3360e442f0.png)

**Comment by WiproPSAccTest** [comment]
Agreed with the resolution , as there is a work around that user can achieve same operation in the Azure portal.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749894
